templates/limit_range.yml

```diff
@@ -0,0 +1,38 @@
+kind: LimitRange
+apiVersion: v1
+metadata:
+  name: limits
+  namespace: "{{ ocp_namespace }}"
+  labels:
+    tenant.paas.redhat.com/tenant: "{{ container_build_tenant }}"
+spec:
+  limits:
+#    - type: openshift.io/ImageStream
+#      max:
+#        openshift.io/image-tags: 1k
+#        openshift.io/images: 1k
+    - type: Pod
+      max:
+        cpu: "{{ cpu_max }}"
+        memory: "{{ memory_max }}"
+      min:
+        cpu: 5m
+        memory: 6Mi
+    - type: Container
+      max:
+        cpu: "{{ cpu_max }}"
+        memory: "{{ memory_max }}"
+      min:
+        cpu: 5m
+        memory: 4Mi
+#      default:
+#        cpu: 500m
+#        memory: 512Mi
+#      defaultRequest:
+#        cpu: 25m
+#        memory: 256Mi
+#      maxLimitRequestRatio:
+#        cpu: '100'
+    - type: openshift.io/Image
```

Comment:
not sure if we will even need this

the only limits we need are for pods I think

Reply:
I am not sure either, but since I could not test limitranges yet I left it in there for now

Reply:
ok leave it there for now then